procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,10 +5,10 @@
    "id": "3dbf3c0c",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    if not (corrections.m1 and corrections.hexapod):
</pre><p>same question as above...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
To check wether they are enabled already or not? If so, it doesn't send the enable command.

Reply:
<p>OK. Back to a less verbose -more concise version of this, just the command and the time when it was executed. </p>